content/ai/concepts/ai-models/models/ai-diffusion-models/ai-diffusion-models.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: 'Diffusion Models'
+Description:'Diffusion Models are generative models, which means they are used to generate data similar to what they were trained on. The models work by destroying training data through the addition of Gaussian noise, and then learning to recover that data.' 
+Subjects:
+  - 'Machine Learning'
+  - 'Data Science'
+  - 'Large Language Model'
```

Comment:
```suggestion
```
Large language model is not one of the subjects in the [subjects.md](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) file.

Reply:
done